File: iOSDevGuide/chapter07/7.2.2/test722/test722/ViewController.m
//
//  ViewController.m
//  test722
//
//  Created by aa64mac on 10/11/15.
//  Copyright © 2015 cyj. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    UITextView *textView = [UITextView new];
    textView.frame = self.view.bounds;
    textView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
    //textView.editable = NO;
    
//    textView.backgroundColor = [UIColor blackColor];
//    textView.textColor = [UIColor whiteColor];
    textView.font = [UIFont systemFontOfSize:32];
    textView.text = @"学习UITextView!\n"
                    "第2行\n"
                    "第3行\n"
                    "第4行\n"
                    "第5行\n"
                    "第6行\n"
                    "第7行\n"
                    "第8行\n"
                    "第9行\n"
                    "第10行\n"
                    "第11行\n"
                    "第12行\n";
    [self.view addSubview:textView];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: iOSDevGuide/chapter08/8.4/Swift-UILabel-Example/Swift-UILabel-Example/ViewController.swift
//
//  ViewController.swift
//  Swift-UILabel-Example
//
//  Created by aa64mac on 10/18/15.
//  Copyright © 2015 cyj. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        let myLabel: UILabel = UILabel()
        myLabel.frame = CGRectMake(0, 0, 300, 100)
        myLabel.layer.position = CGPoint(x: self.view.bounds.width / 2, y: 200)
        print("position:\(myLabel.layer.position)")
        myLabel.backgroundColor = UIColor.redColor()
        myLabel.text = "Hello!"
        myLabel.font = UIFont.systemFontOfSize(40)
        myLabel.textColor = UIColor.whiteColor()
        myLabel.shadowColor = UIColor.blueColor()
        myLabel.textAlignment = NSTextAlignment.Center
        myLabel.layer.masksToBounds = true
        myLabel.layer.cornerRadius = 20.0
        self.view.addSubview(myLabel)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

